src/modules/commentsApiConfig.js

```diff
@@ -0,0 +1,64 @@
+import countComments from './commentsCounter.js';
+
+const COMMENTS_URL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Nbc4s2TFp5CN6zapCfwg/comments?item_id=';
+const commentsContainer = document.querySelector('.comments');
+
+const getSpecificComment = async (id) => {
+  const response = await fetch(`${COMMENTS_URL}${id}`);
+  const data = await response.json();
+  return data;
+};
+
+const postComment = async (id, name, comment) => {
+  const response = await fetch(COMMENTS_URL, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: id,
+      username: name,
+      comment,
+    }),
+  });
+  return response.json();
+};
```

Comment:
- [OPTIONAL] It would be better when a user clicks the "Comment" button to submit the author's comment, the data should be saved in [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) and directly update the screen instead of closing the comment popup page and reopening it to match the change. 👍🏻